hbase-hadoop-compat/src/main/java/org/apache/hadoop/metrics2/lib/DynamicMetricsRegistry.java

```diff
@@ -532,4 +534,9 @@ public void clearMetrics() {
     }
     metricsMap.clear();
   }
+
+  @VisibleForTesting
```

Comment:
We do not allow VisibleForTesting annotation now.

Please use RestrictedApi to restrict the usage.

Reply:
ok thx

Reply:
@Apache9 I've already changed it, pls help me review, thx a lot!